### ISSUE TITLE
Multi-threading NEAT

### ISSUE THREAD

**Issue by jelisavetailic** [ncsa/NEAT#9]
Hi, 

I saw in your changeLogs this:

For now, we've eliminated the "Jobs" option and the merge jobs function. We plan to implement multi-threading instead as a way to speed up NEAT's simulation process.

Could you tell me if multi-threading is implemented or how can I speed up NEAT?

Thank you!

**Comment by joshfactorial** [comment]
We're investigating re-implementing the function. The easiest thing to do for now would be just to split the input fasta by contig, run NEAT in parallel on each resultant file, then cat the results back together (or use samtools for the bams).

**Comment by jelisavetailic** [comment]
Thank you for your answer!

**Comment by joshfactorial** [comment]
In the current method, I keep paired ends together during processing. I'm not sure what all problems multithreading might cause, which is part of why I put it on hold.
